roles/basic_setup_rhel/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: "Subscription Manifest"
+  include_role:
+    name: theforeman.foreman.manifest
+
+- name: "Enable RHEL7 repository"
+  include_role:
+    name: theforeman.foreman.repositories
+  vars:
+    foreman_products:
+      - name: Red Hat Enterprise Linux Server
+        repository_sets:
+          - name: Red Hat Enterprise Linux 7 Server (RPMs)
+            basearch: x86_64
```

Comment:
THe readme should point out that this does x86_64 only.

Reply:
We can always add the option to enable other architectures later if it's something that users want.